schema/cam/cam_schema_2-1-0.json

```diff
@@ -126,308 +128,829 @@
+                        "type": "object",
+                        "description": "offset position of a detected event point with regards to the previous detected event point (reference_position).",
+                        "properties": {
+                          "delta_latitude": {
+                            "$comment": "if not provided, 'delta_latitude' = 131072 (unavailable)",
+                            "type": "integer",
+                            "description": "oneMicrodegreeNorth (10), oneMicrodegreeSouth (-10) , unavailable(131072)",
+                            "default": 131072,
+                            "minimum": -131071,
+                            "maximum": 131072
+                          },
+                          "delta_longitude": {
+                            "$comment": "if not provided, 'delta_longitude' = 131072 (unavailable)",
+                            "type": "integer",
+                            "description": "oneMicrodegreeEast (10), oneMicrodegreeWest (-10), unavailable(131072)",
+                            "default": 131072,
+                            "minimum": -131071,
+                            "maximum": 131072
+                          },
+                          "delta_altitude": {
+                            "$comment": "if not provided, 'delta_altitude' = 12800 (unavailable)",
+                            "type": "integer",
+                            "description": "oneCentimeterUp (1), oneCentimeterDown (-1), unavailable(12800)",
+                            "default": 12800,
+                            "minimum": -12700,
+                            "maximum": 12800
+                          }
+                        }
                       },
-                      "delta_altitude": {
-                        "$comment": "if not provided, 'delta_altitude' = 12800 (unavailable)",
+                      "path_delta_time": {
                         "type": "integer",
-                        "description": "oneCentimeterUp (1), oneCentimeterDown (-1), unavailable(12800)",
-                        "default": 12800,
-                        "minimum": -12700,
-                        "maximum": 12800
+                        "description": "time travelled by the detecting ITS-S since the previous detected event point (generation_delta_time). tenMilliSecondsInPast(1)",
+                        "minimum": 1,
+                        "maximum": 65535
                       }
                     }
-                  },
-                  "path_delta_time": {
-                    "type": "integer",
-                    "description": "time travelled by the detecting ITS-S since the previous detected event point (generation_delta_time). tenMilliSecondsInPast(1)",
-                    "minimum": 1,
-                    "maximum": 65535
                   }
                 }
               }
             }
           }
+        },
+        "special_vehicle_container": {
```

Comment:
`special_vehicle_container` is a new item; it should go in its own set of commits, one per new entry in it, rather than be conflated in a commit that fixes the existing schema.

It would also make reviewing easier by splitting the changes in smaller, easier-to-digest changes.

Reply:
Ok, done